prog/weave-kube/weave-daemonset-k8s-1.6.yaml

```diff
@@ -92,6 +92,11 @@ spec:
               cpu: 10m
           securityContext:
             privileged: true
+          env:
+          - name: MY_NODE_NAME
```

Comment:
I am thinking `MY_NODE_NAME`'s name may lead people to think it isn't something actually used, or something which should be replaced with another name.
Maybe something like `WEAVE_NPC_NODE_NAME` would be more explicit and show it actually has a purpose as it stands?